Checking the Genuineness of your Skycoin Hardware Wallet.md

```diff
@@ -0,0 +1,54 @@
+# Checking the Genuineness of your Skycoin Hardware Wallet
+
+## Checking for Physical Issues
+
+Once you receive the shipment, please check for any tampering or damage of the package.
+
+<A Picture of an untampered shipment before opening>
+
+Upon opening the package, please ensure, the holographic sticker with the Skycoin logo on the box is not broken.
+<A Picture of the unbroken holographic seal on the box>
+
+## Checking for Software/Technical Issues
+
+Skycoin Hardware Wallet provides maximum security to the user.
+
+However, it is always important to check whether your Skycoin Hardware Wallet is not somehow compromised before adding your coins to the wallet.
```

Comment:
Delete "somehow".  Please use "...before using the device". Delete always. Never use wallet for the Skywallet. the wallet is the desktop wallet.